src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2509,6 +2509,11 @@ namespace winrt::TerminalApp::implementation
         CATCH_LOG();
     }
 
+// The current UCRT has a neat AVX2 implementation for all of the various strlen() functions.
+// This greatly benefits _PasteFromClipboardHandler, where we're calling wcslen while holding a global
+// lock onto the clipboard. MSVC treats wcslen as a builtin and so we need to disable it explicitly.
+#pragma function(wcslen)
```

Comment:
_what,_ this is terrifying.

Reply:
does this compile with clang?

Reply:
Do you mean the clang-cl kind? As far as I can tell, clang never uses an intrinsic `wcslen` in the first place (which is nice, because why would it). For instance: https://godbolt.org/z/7rezdsWd9